utils/restutil/view/view.go

```diff
@@ -41,20 +41,34 @@ type RESTView struct {
 }
 
 func (p *RESTView) RenderSuccessPost(w rest.ResponseWriter, r *rest.Request, id string) {
-	w.Header().Add(HttpHeaderLocation, fmt.Sprintf("./%s/%s", strings.TrimLeft(r.URL.Path, "/api/0.0.1/"), id))
+	w.Header().Add(
+		HttpHeaderLocation,
+		fmt.Sprintf("./%s/%s", strings.TrimLeft(r.URL.Path, "/api0.1"), id),
+	)
```

Comment:
This TrimLeft looks wrong (and it already was before your change).
```suggestion
	w.Header().Add(
		HttpHeaderLocation,
		fmt.Sprintf(".%s/%s", strings.TrimPrefix(r.URL.Path, "/api/0.0.1"), id),
	)
```

Reply:
If you see the force-pushed in the PR, I tried first using `TrimPrefix` but several unit tests failed. I tried with slight variations of adding a `/` in the format or in the prefix, but couldn't get all tests happy.

Can you try on your end? Maybe is just the tests that have the wrong expectations?

Reply:
Fixed using `fmt.Sprintf(".%s/%s", strings.TrimPrefix(strings.TrimPrefix(r.URL.Path, "."), "/api"), id)`